features/upgrade/storage/upgrade.feature

```diff
@@ -344,3 +344,120 @@ Feature: Storage upgrade tests
       | ls | /mnt/hostpath |
     Then the output should contain:
       | test-before-upgrade |
+
+  # @author ropatil@redhat.com
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: Snapshot operator should work well before and after upgrade of cluster on AWS platform - prepare
```

Comment:
I see it in polarion it's a new case. The scenario named `"[AWS-EBS] [Snapshot operator] should work well before and after upgrade of a cluster"` is better.